README.md

```diff
@@ -66,6 +66,8 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 
 > **NOTE:** PyTorch 2.2.1 does not support `torch.compile` with Python 3.12. On Fedora 39+, install `python3.11-devel` and create the virtual env with `python3.11` if you wish to use PyTorch's JIT compiler.
 
+> **NOTE** When installing the `ilab` CLI on macOS, you may have to run the `xcode select --install` command to install the required packages previously listed. 
```

Comment:
The linter does not like a blank line between to blockquotes (`>`). If you put `<!-- -->` (an HTML comment) on the blank line, you get what you want and the linter does not complain.